docs/src/development.md

```diff
@@ -319,7 +319,13 @@ For PRs triggered from branches inside the Trixi.jl main repository previews of
 the new documentation are generated at `https://trixi-framework.github.io/Trixi.jl/previews/PRXXX`,
 where `XXX` is the number of the PR.
 This does not work for PRs from forks for security reasons (since anyone could otherwise push
-arbitrary stuff to the Trixi website, including malicious code).
+arbitrary stuff to the Trixi website, including malicious code). In order to build the documentation
+locally for PRs from forks you can use the following commands that mimic what is done by the CI
+```bash
+julia --project=docs -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+julia --project=docs --color=yes docs/make.jl
+```
+where, again, the html files are found in the `docs/build` folder.
```

Comment:
This contradicts the part written above. Could you please update/replace it?

Reply:
Do you mean just replace the original command with this new one?

Reply:
Yes. I find it quite confusing to have two slightly different descriptions to achieve the same goal where only one of them works reliably.